tutorials/phase_contrast_calculation.ipynb

```diff
@@ -0,0 +1,216 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Phase Contrast (PC) calculation tutorial"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Start by importing the required libraries and defining some settings:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from tabulate import tabulate\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "from ukat.data import fetch\n",
+    "from ukat.vessels.phase_contrast import PhaseContrast\n",
+    "\n",
+    "# Ensure figures are rendered in the notebook\n",
+    "%matplotlib inline\n",
+    "\n",
+    "# Initialise output path for PC outputs\n",
+    "directory = 'phase_calculation_output'\n",
+    "os.makedirs(directory, exist_ok=True)\n",
+    "OUTPUT_DIR = os.path.join(os.getcwd(), directory)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Retrieve PC test data using our data fetcher. The first time the fetcher is run, it will download the data from the\n",
+    "internet, after that it will just use the cached local data.\n",
+    "\n",
+    "This will return 5 variables:\n",
+    "* `magnitude`: a 3D array with the magnitude data at all trigger delay times (phase).\n",
+    "* `vel_array`: a 3D array with the phase data at all trigger delay times (phase).\n",
+    "* `affine`: the affine matrix for the data (needed to later write a new NIfTI file).\n",
+    "* `mask`: a 3D boolean matrix with the pixels to fit/calculate.\n",
+    "* `velocity_encoding`: returns a float values of the velocity encoding in cm/s"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Fetch test data\n",
+    "magnitude, vel_array, mask, affine, velocity_encoding = fetch.phase_contrast_left_philips()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Calculate velocity and flow (RBF) parameters using the `PhaseContrast` class implemented in `vessels`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Initialise Phase Contrast calculation object\n",
+    "pc_obj = PhaseContrast(vel_array, affine, mask=mask)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Plot the **velocity** and the **rbf** per trigger delay time (also known as phase)."
```

Comment:
Maybe capitalise RBF. I know its not capitalised as the variable, but it still seems better to use correct grammar in plain text.